image.yaml

```diff
@@ -37,12 +37,12 @@ modules:
                   ref: 0.45.4
           - name: jboss-eap-modules
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_7410_CR2
+                  url: https://github.com/rdnovell/jboss-eap-modules.git
```

Comment:
Your fork.

Reply:
Done in commit [99b9035](https://github.com/rdnovell/jboss-eap-7-openshift-image/commit/99b9035eb1e3920e685ae97900f9b0cd334b448d)